src/react/hooks/useBackgroundQuery.ts

```diff
@@ -170,7 +170,19 @@ export function useBackgroundQuery<
   UseBackgroundQueryResult<TData, TVariables>,
 ];
 
-export function useBackgroundQuery<
+export function useBackgroundQuery() {
+  // @ts-expect-error Cannot assign to 'useBackgroundQuery' because it is a function.ts(2630)
+  useBackgroundQuery = makeHookWrappable(
+    "useBackgroundQuery",
+    (_, options) =>
+      useApolloClient(typeof options === "object" ? options.client : undefined),
+    _useBackgroundQuery as any
+  );
+
+  return useBackgroundQuery.apply(null, arguments as any);
```

Comment:
Perhaps I'm missing something obvious or its been a long travel day, but couldn't you just set this to any variable name and call it? Any reason you need to override the original function?

```suggestion
  const uBQ = makeHookWrappable(
    "useBackgroundQuery",
    (_, options) =>
      useApolloClient(typeof options === "object" ? options.client : undefined),
    _useBackgroundQuery as any
  );

  return uBQ.apply(null, arguments as any);
```

or if if you wanted to do it all inline

```ts
return makeHookWrappable(
  "useBackgroundQuery",
  (_, options) =>
    useApolloClient(typeof options === "object" ? options.client : undefined),
  _useBackgroundQuery as any
).apply(null, arguments as any);
```